lib/tapioca/dsl/compiler.rb

```diff
@@ -89,7 +88,7 @@ def all_modules
             if @@requested_constants.any?
               @@requested_constants.select { |k| k.is_a?(Module) }
```

Comment:
May as well change this too, to be consistent.

```suggestion
              @@requested_constants.grep(Module)
```